packages/react-native/React/CxxBridge/RCTMessageThread.h

```diff
@@ -12,6 +12,7 @@
 
 #import <React/RCTJavaScriptExecutor.h>
 #import <cxxreact/MessageQueueThread.h>
+#import <atomic>
```

Comment:
The error occurs on line 36. `std::atomic_bool` is not valid without first importing `atomic`.